### PR TITLE
skip headroom pool test for Force10-S6100 T1 topo

### PR DESCRIPTION
**Pull request by XuChen-MSFT**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [x] 202305
- [x] 202311

#### What is the motivation for this PR?

6100 T1's headroom pool test fail
For 6100, T0 and T1 have same buffer configuration, and T1's failure is related to lag keepalive.
xfail test for 6100's T1 topology.
and 6100's headroom pool test is covered by T0.

#### How did you do it?

set xfail for 6100 T1's headroom pool testcase

### CODE REVIEW

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Comment by StormLiangMS** [comment]
@XuChen-MSFT do we plan to fix it in future? Or we will skip this case for T1-lag topo? If we plan to fix, pls add an issue. If not, let's skip it other than xfail.

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -1146,6 +1150,7 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolWatermark:
   xfail:
     reason: "Headroom pool size not supported."
     conditions:
+      - "hwsku in ['Force10-S6100'] and topo_type in ['t1-64-lag']"
```
Comment (ZhaohuiS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1153]: @XuChen-MSFT for those 2 conditions under testQosSaiHeadroomPoolWatermark case, the relationship between them is  AND or OR, in my understanding, it's OR, right? Miss the following line?

conditions_logical_operator: or

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -1146,6 +1150,7 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolWatermark:
...
       - "hwsku not in ['Arista-7060CX-32S-C32', 'Celestica-DX010-C32', 'Arista-7260CX3-D108C8', 'Force10-S6100', 'Arista-7260CX3-Q64', 'Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8']"
```
Comment (neethajohn) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1154]: these should should be skipped not xfail (as is done for testQosSaiHeadroomPoolSize

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml (hunk 1 above)
Comment (XuChen-MSFT) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: filed github issue #12292

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml (hunk 2 above)
Comment (XuChen-MSFT) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: updated to "skip"

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**